Fresh `yarn add` doesn't work

**Describe the bug**

It results in a `Error: Cannot find module 'ganache-core'`

**Expected behavior**
It should just work when installing from npm

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Specifications**
 - Version:
 - OS:
 - Node Version:

**Additional context**
Add any other context about the problem here.

--- a/package.json
+++ b/package.json
@@ -30,7 +30,8 @@
     "@ethersproject/contracts": "^5.0.5",
     "@ethersproject/hardware-wallets": "^5.0.8",
     "@openzeppelin/contracts": "^3.3.0",
-    "ethers": "5.0.0"
+    "ethers": "5.0.0",
+    "ganache-core": "^2.12.1"
   },
   "devDependencies": {
     "@eth-optimism/smock": "^0.0.2",
@@ -49,7 +50,6 @@
     "copyfiles": "^2.3.0",
     "ethereum-waffle": "3.0.0",
     "fs-extra": "^9.0.1",
-    "ganache-core": "^2.12.1",
     "lodash": "^4.17.20",
     "merkle-patricia-tree": "^4.0.0",
     "mkdirp": "^1.0.4",

